barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.cpp

```diff
@@ -338,6 +339,7 @@ bool ClientIVC::verify(const Proof& proof)
 HonkProof ClientIVC::decider_prove() const
 {
     vinfo("prove decider...");
+    fold_output.accumulator->proving_key.commitment_key = bn254_commitment_key;
```

Comment:
I didn't want to do this but IIRC a bigger refactor would be needed to construct the initial accumulator with this key (it starts out as a nullptr).